packages/wonder-blocks-icon/components/icon.js

```diff
@@ -77,7 +77,14 @@ export default class Icon extends React.PureComponent<Props> {
     };
 
     render() {
-        const {"aria-label": ariaLabel, color, icon, size, style} = this.props;
+        // There is a weird thing where Flow will only recognize a string-quoted
+        // prop name if it's in single quotes, but our tooling normalizes it to
+        // double-quotes on commit. So the aria-label prop isn't included in
```

Comment:
@wishdasher thanks for figuring that out.  That's such a bizarre bug.  I assumed flow was converting to an AST before doing its checks.